src/utils/regExp.ts

```
@@ -0,0 +1 @@
+export const mailRegExp = /^[a-zA-Z0-9+-\_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/
```

ESLint

[error] Unnecessary escape character: \_.

(no-useless-escape)